content/en/docs/components/training/job-scheduling.md

```diff
@@ -75,6 +75,8 @@ In installing Scheduler Plugins as a default scheduler, you don't need to specif
 You have to install volcano scheduler in your cluster first as a secondary scheduler of Kubernetes and
 configure operator to select the scheduler name for gang-scheduling in the following:
 
+- training-operator
```

Comment:
> Why is this mentioning the training operator?

@alculquicondor Does that mean this line?

The `job-scheduling.md` is not docs only for the mpi-operator. However, we have not had docs on how to set up gang-scheduling in the mpi-operator and only have the docs for the training-operator.

So I added new docs for the mpi-operator since the training-operator and the mpi-operator have different ways of setting up the gang-scheduling.

Reply:
ahh, I missed `mpi-operator` below.